curvefs/src/mds/topology/topology.cpp

```diff
@@ -537,6 +540,7 @@ TopoStatusCode TopologyImpl::AddPartition(const Partition &data) {
     ReadLockGuard rlockPool(poolMutex_);
     WriteLockGuard wlockCopyset(copySetMutex_);
     WriteLockGuard wlockPartition(partitionMutex_);
+    WriteLockGuard wlockCluster(clusterMutex_);
```

Comment:
lock seq
use pool lock ?

Reply:
The order of  taking the clusterMutex_  lock here does not matter.